.github/workflows/metadata-refresh.yml

```diff
@@ -0,0 +1,68 @@
+name: Metadata Refresh
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'The environment to run the workflow in'
+        required: true
+        type: choice
+        default: 'preview'
+        options:
+          - 'preview'
+          - 'production'
+  # schedule:
+  #   - cron: '0 0 * * *' # Run at midnight UTC
+
+jobs:
+
+  metadata_refresh:
+    name: Metadata Refresh
+    runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment || github.event_name == 'schedule' && 'production' || 'preview' }}
+    env:
+      AWS_S3_BUCKET: "${{secrets.HUB_METADATA_S3_BUCKET }}"
+    permissions:
+      id-token: write # This is required for requesting the JWT
+    steps:
+    - uses: actions/checkout@v3.3.0
+      # TODO: will need this to trigger CI jobs
+      # with:
+      #   token: ${{ secrets.GH_PAT }}
```

Comment:
I don't understand this comment. Is it stale? Why would we need a personal access token to checkout the repository, and what does that have to do with triggering CI jobs?

Reply:
Yep thats stale, I thought I had removed it.